website/versionsArchived.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
FYI @lex111 if you archive a version you can get an immutable Netlify deployment link and add it there. I don't think it's needed to keep versions 3 months now, we can just add older versions this way in a more exhaustive way

Reply:
That's nice, thanks, maybe we can create [npm/bash] command to automate this? And why are URLs so long?

Reply:
> maybe we can create [npm/bash] command to automate this

Ideally, I'd like to have an official `docusaurus archive` CLI but that's not so simple due to the plugin system, multi-instance etc... 

> And why are URLs so long?

Those are the unique deploy URLs from Netlify. I don't think it's worth creating a custom subdomain DNS for all the older versions. I don't have direct access to the DNS config so it's a bit annoying to have to ask Joel on each archived release. Note also the React doc does this, some older docs redirect to an immutable vercel deployment: https://reactjs.org/versions